apps/federation/lib/DbHandler.php

```diff
@@ -87,7 +87,7 @@ public function addServer($url) {
 		$result = $query->execute();
 
 		if ($result) {
-			return (int)$this->connection->lastInsertId('*PREFIX*'.$this->dbTable);
+			return $this->connection->lastInsertId('*PREFIX*'.$this->dbTable);
```

Comment:
https://github.com/nextcloud/server/blob/master/lib/private/DB/Connection.php#L273-L289

This is not typed to integer... Only the querybuilder one is

Reply:
But since this uses the query builder, just replace it with:
```suggestion
			return $query->getLastInsertId();
```